compiler/quilt/test/test_command.py

```diff
@@ -19,6 +61,14 @@
 from .utils import QuiltTestCase, patch
 
 class CommandTest(QuiltTestCase):
+    def _mock_method(self, endpoint, status=201, team=None, message="", method=responses.POST, api="users"):
```

Comment:
It seems like it's only useful for error responses, since anything else would return more than just `message=...`. I'd call it `_mock_error`. Other things:
- We don't really have an "api" vs "endpoint"; make it just a single "endpoint" param. (Also, "users" isn't good as a default cause it's just one of the many APIs.)
- Make the status default to some error code - or just make it required.